metabase/metabase
# Use new ColorSelector for single color chart settings

Epic https://github.com/metabase/metabase/issues/22816

How to test:
- New question -> Orders, group by Created At, Sum by Count
- Visualization -> Line -> Display
- Visualization -> Waterfall -> Display
- Visualization -> Table -> Conditional formatting -> Add rule
- New question -> Orders, Sum by Count (without grouping)
- Visualization -> Progress -> Display
- Visualization -> Gauge -> Display
- Make sure it is possible to set colors

Please note that the real color palette would have 3 times more colors, that's why we are showing 9 colors per row.

<img width="628" alt="Screenshot 2022-05-26 at 18 18 45" src="https://user-images.githubusercontent.com/8542534/170519753-00936b7e-4959-4b1c-9037-623753f887be.png">
<img width="619" alt="Screenshot 2022-05-26 at 18 19 35" src="https://user-images.githubusercontent.com/8542534/170519776-56d80042-2deb-4d27-8fc5-7928ffe144b0.png">
<img width="765" alt="Screenshot 2022-05-26 at 18 20 04" src="https://user-images.githubusercontent.com/8542534/170519782-d08b3bba-ebf5-4e3b-adb1-6d8ec4f2ed21.png">
<img width="636" alt="Screenshot 2022-05-26 at 18 20 37" src="https://user-images.githubusercontent.com/8542534/170519786-1085f74f-0fdd-4491-838d-36615998edb6.png">
<img width="992" alt="Screenshot 2022-05-26 at 18 20 52" src="https://user-images.githubusercontent.com/8542534/170519790-5ce78960-1330-41de-ab03-b533065a2339.png">
